### ISSUE TITLE
Getting "ERROR(6006)" even though I just downloaded it?

### ISSUE THREAD

**Issue by at-a-minimum1** [ShikyoKira/Project-New-Reign---Nemesis-Main#472]
Checking engine version
ERROR(6006): Missing latest engine files. Update engine to fix it

Failed to generate behavior

So I hit update engine and the program closes. Was also having issues running it period without it closing. I fixed that by toggling "Run Detached" off in Vortex. I've tried running it through the file itself, but that doesn't work either. It could be user error, but I don't see where I'm going wrong. Any advice? Thanks. 

![image](https://user-images.githubusercontent.com/70494187/107143580-d38d4000-6903-11eb-991f-52e684c0ad91.png)

**Comment by at-a-minimum1** [comment]
I figured out my issue and then found another. 

... Figured out that issue too. 

It was user error. I forgot to install the mod through Nexus. It was in my download folder and I forgot to hit install. Then I had an issue with the "NEXUS BACKUP FILES" (Error 2024). So I went through and deleted all of those files. Now it works. Thanks so much. :) 

This is the page I used for the second issue. https://github.com/ShikyoKira/Project-New-Reign---Nemesis-Main/issues/467
